src/components/Detail/Detail.tsx

```diff
@@ -1,15 +1,16 @@
-import { ComponentProps, forwardRef } from 'react';
+import React, { ComponentProps, forwardRef } from 'react';
 import { getCommonProps } from '../../utils';
+import classNames from 'classnames';
 
 export interface DetailProps extends ComponentProps<'div'> {
   /*
    * Label that appears on the left side of the Detail component
    */
-  label: string;
+  label?: React.ReactNode;
```

Comment:
if we are using `label` and `value` to derive the key, they should be required

otherwise, should make `key` required

Reply:
yeah I would argue detail without a label and value doesn't make sense as well

Reply:
are there any cases where you'd have one but not the other?